scripts/romanisim-make-stack

```diff
@@ -0,0 +1,267 @@
+                        default=False,
+                        help=('RA & Dec specifies location of boresight, not center of WFI.'))
+
+    # WCS Object Catalog
+    parser.add_argument('-c', '--catalog',
+                        type=str,
+                        metavar='object_table.csv',
+                        default=None,
+                        help='Object catalog file for wcs matching (csv)')
+
+    # Debug argument
+    parser.add_argument('-d', "--debug",
+                        action='store_true',
+                        help='Display verbose debug output')
+
+    # Level argument
+    parser.add_argument('-l', '--level',
+                        type=int,
+                        default=1,
+                        help='1 or 2, for L1 or L2 output')
+
+    # Create script making argument
+    parser.add_argument('-m', "--make_script",
+                        action='store_true',
+                        help='Create list of romanisim calls from csv file (e.g. for cluster usage).')
+
+    # Number of objects to simulate
+    parser.add_argument('-n', '--nobj',
+                        type=int,
+                        default=1000,
+                        help='Specify number of objects to simulate.')
+
+    # Persistence
+    parser.add_argument('-p', '--previous',
+                        type=str,
+                        metavar='previous.asdf',
+                        nargs=1,
+                        help=(
+                            'Previous simulated file in chronological order used for persistence modeling.'))
+
+    # Random number seed
+    parser.add_argument('-r', '--rng_seed',
+                        type=int,
+                        default=None,
+                        help='Random number seed (int)')
+
+    # SCA detector identifier number
+    parser.add_argument('-s','--sca',
+                        type=int,
+                        default=0,
+                        help='SCA to simulate')
+
+    # Use CRDS for distortion maps
+    parser.add_argument('-u', '--usecrds',
+                        action='store_true',
+                        help='Use CRDS for distortion map')
+
+    # Use CRDS stored PSFs
+    parser.add_argument('-w', '--webbpsf',
+                        action='store_true',
+                        help='Use webbpsf for PSFs.')
+
+
+
+
+    args = parser.parse_args()
+
+    log.info('Starting simulation...')
+    log.warning("romanisim is under active development.  Its output has "
+                "not been formally validated; only limited testing has been "
+                "performed.  For this reason, use of romanisim for "
+                "preparation of ROSES proposals is not advised.  Other "
+                "packages like galsim's roman package or STIPS may better "
+                "serve such purposes.")
+
+    # Set up metadata
+    metadata = deepcopy(default_parameters_dictionary)
+
+    # If creating a script, create the script file
+    if args.make_script:
```

Comment:
Maybe for this mode, the argument should be
--script
and defaults to None (don't make a script), and otherwise is the name of the file to make?

Reply:
It does default to None (technically False) by not being activated - the reason it is what it is is in order to have '-m'.

Reply:
My point is really to allow this argument to do double duty as the name of the script to make (if one wants a script), rather than be true / false.

Reply:
Implemented.